web/src/components/pages/profile/avatar-setup/avatar-setup.css

```diff
@@ -1,8 +1,64 @@
 @import url('../../../vars.css');
+.full-avatar-setup {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    max-width: var(--wide-desktop-width);
+}
+
+.clip {
+    & h2 {
+        width: 334px;
+        height: 58px;
+        font-family: ZillaSlab;
```

Comment:
can we use our css variables for fonts here `--base-font-family` and `--strong-font-family` if I remember correctly

Reply:
:+1: